apps/kevin-malone/bot/config.py

```diff
@@ -37,12 +37,12 @@
 WALLET_VERIFICATION_INSTRUCTIONS = (
     "To verify you have access to the wallet at your address, follow these steps:"
     "\n1: Navigate to myetherwallet.com/wallet/sign"
-    "\n2: When prompted, connect with your metamask wallet in the browser"
+    "\n2: Connect to MyEtherwallet with your wallet in the browser by pressing \"browser extension\""
```

Comment:
Super minor nitpick but wonder if we want the casing to match what they use on the site -- MyEtherWallet.